src/resources/language/_language-zh.yml

```diff
@@ -0,0 +1,62 @@
+toc-title-document: "目录"
+toc-title-website: "在该页面"
```

Comment:
^ Not entirely sure whether this translation matches the context of 'toc-title-website' -- Can I please see what it would look like in the UI?

Reply:
Here is the usage (text above toc at right):

<img width="1166" alt="Screen Shot 2021-11-08 at 9 14 04 PM" src="https://user-images.githubusercontent.com/104391/140852081-ef52aae9-66a0-4068-b1bf-17cb6623c29b.png">

Reply:
OK. Revised this translation to say something more sensible.